tests/src/com/unciv/logic/city/CityConquestFunctionsTest.kt

```diff
@@ -103,7 +103,7 @@ class CityConquestFunctionsTest {
     @Test
     fun `should plunder more gold with building uniques upon capture`() {
         // given
-        val building = testGame.createBuilding(UniqueType.DoublesGoldFromCapturingCity.text)
+        val building = testGame.createBuilding(UniqueType.GoldFromCapturingCity.text)
```

Comment:
As the failing check says - that won't work. The old Unique had no parameters and works out of the box. The new one has, and needs parseable values for them, don't leave the placeholders. There should be an easy function somewhere...
```suggestion
        val building = testGame.createBuilding(UniqueType.GoldFromCapturingCity.text.fillPlaceholders("+100"))
```
... but 't would be better you test such stuff locally.